samza-sql/src/main/java/org/apache/samza/sql/translator/QueryTranslator.java

```diff
@@ -151,7 +154,10 @@ public RelNode visit(LogicalAggregate aggregate) {
 
     Optional<TableDescriptor> tableDescriptor = sinkConfig.getTableDescriptor();
     if (!tableDescriptor.isPresent()) {
-      outputStream.sendTo(streamGraph.getOutputStream(sinkConfig.getStreamName()));
+      KVSerde<Object, Object> noOpKVSerde = KVSerde.of(new NoOpSerde<>(), new NoOpSerde<>());
```

Comment:
Is this the expected final implementation for this?

Reply:
No, I need to check with Aditya/Srini about how serdes are handled in sql.

Reply:
This is the correct final implementation. SQL does serde itself (using SamzaRelConverter) within the DAG, so the framework shouldn't be doing any Serde.